recipes/spam/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "spam" %}
+{% set version = "0.6.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gricad-gitlab.univ-grenoble-alpes.fr/ttk/spam/-/archive/62a7c325dbea0d90314ba19a0ac76a61a23691f4/spam-62a7c325dbea0d90314ba19a0ac76a61a23691f4.tar
+  sha256: bc1c898392eac220784ae534550e2b3b0c071053f9aca870264cc79718b8886e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python       # [unix]
+    - python =3.7  # [win]
+    - numpy
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
i think you'd want to hold off on the cross-compiling and wait for the bot to apply the current best-practice changes.

Otherwise: this build env looks pretty heavy: it may be a fair amount of this belongs only in host, but i don't know enough to say.

There is also the `pybind11_abi` which seems to be important to other packages getting correct .

Reply:
We've removed cross compiling and added `pybind11_abi`

Reply:
Actually we put back `cross-python_{{ target_platform }}` as it seems to solve the apparently classical error
```
[noarch_python] Noarch package contains binary script
```
[log](https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/471182/logs/19 )